decidim-core/lib/decidim/core/engine.rb

```diff
@@ -55,6 +55,168 @@ class Engine < ::Rails::Engine
+        Decidim.icons.register(name: "documents", icon: "file-text-line", description: "Document", category: "documents", engine: :core)
+        Decidim.icons.register(name: "folder_open", icon: "folder-open-line", description: "Folder open", category: "documents", engine: :core)
+        Decidim.icons.register(name: "folder_close", icon: "folder-line", description: "Folder close", category: "documents", engine: :core)
+        Decidim.icons.register(name: "document_weight", icon: "scales-2-line", description: "Doc. weight (kb/mb)", category: "documents", engine: :core)
+        Decidim.icons.register(name: "document_download", icon: "download-line", description: "Download", category: "documents", engine: :core)
+        Decidim.icons.register(name: "images", icon: "image-line", description: "Images", category: "documents", engine: :core)
+        Decidim.icons.register(name: "all", icon: "apps-2-line", description: "All", category: "other", engine: :core)
+        Decidim.icons.register(name: "other", icon: "question-line", description: "Other", category: "other", engine: :core)
+
+        # social icons
+        Decidim.icons.register(name: "wechat-line", icon: "wechat-line", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "linkedin-box-fill", icon: "linkedin-box-fill", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "twitter-x-line", icon: "twitter-x-line", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "whatsapp-line", icon: "whatsapp-line", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "telegram-line", icon: "telegram-line", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "twitter-x-fill", icon: "twitter-x-fill", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "google-fill", icon: "google-fill", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "facebook-fill", icon: "facebook-fill", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "instagram-line", icon: "instagram-line", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "youtube-line", icon: "youtube-line", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "github-fill", icon: "github-fill", category: "social icon", description: "", engine: :core)
+        Decidim.icons.register(name: "facebook-circle-line", icon: "facebook-circle-line", category: "social icon", description: "", engine: :core)
+        Decidim.social_share_services_registry.manifests.each do |manifest|
+          Decidim.icons.register(name: manifest.icon, icon: manifest.icon, description: "Social share ", category: "social icon", engine: :core)
+        end
```

Comment:
There's a bug here with some of the icons, that we're using SVGs: 

![Screenshot of the missing icons](https://github.com/decidim/decidim/assets/717367/7c829918-cc82-4409-984e-903a2428bc3e)

Can you fix that please? Ideally we would show these icons here too

Reply:
To fix this, i had to define a "Social Share" section containing only the icons from the social share. Please check 44251c4018996a6db6f6b3f68b5ba06ead3a6f8c